tsl/test/shared/expected/dist_fetcher_type-15.out

```diff
@@ -0,0 +1,271 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+\set ON_ERROR_STOP off
+-- Test that we use the correct type of remote data fetcher.
+set timescaledb.remote_data_fetcher = 'auto';
+select 1 x from distinct_on_distributed t1, distinct_on_distributed t2
+where t1.id = t2.id + 1
+limit 1;
+ x 
+ 1
+(1 row)
+
+-- This query should choose COPY fetcher.
+select 1 x from distinct_on_distributed t1
+limit 1;
+ x 
+ 1
+(1 row)
+
+explain (analyze, verbose, costs off, timing off, summary off)
+select 1 x from distinct_on_distributed t1
+limit 1;
+QUERY PLAN
+ Limit (actual rows=1 loops=1)
+   Output: 1
+   ->  Result (actual rows=1 loops=1)
+         Output: 1
+         ->  Custom Scan (DataNodeScan) on public.distinct_on_distributed t1 (actual rows=1 loops=1)
+               Data node: data_node_1
+               Fetcher Type: COPY
+               Chunks: _dist_hyper_X_X_chunk
+               Remote SQL: SELECT NULL FROM public.distinct_on_distributed WHERE _timescaledb_internal.chunks_in(public.distinct_on_distributed.*, ARRAY[..]) LIMIT 1
+(9 rows)
+
+set timescaledb.remote_data_fetcher = 'cursor';
+select 1 x from distinct_on_distributed t1, distinct_on_distributed t2
+where t1.id = t2.id + 1
+limit 1;
+ x 
+ 1
+(1 row)
+
+explain (analyze, verbose, costs off, timing off, summary off)
+select 1 x from distinct_on_distributed t1, distinct_on_distributed t2
+where t1.id = t2.id + 1
+limit 1;
+QUERY PLAN
+ Limit (actual rows=1 loops=1)
+   Output: 1
+   ->  Nested Loop (actual rows=1 loops=1)
+         Output: 1
+         Join Filter: (t1.id = (t2.id + 1))
+         Rows Removed by Join Filter: 40000
```

Comment:
So do we know why the plan has changed? Looks like the LIMIT stopped working for the distributed tables, this is probably a bug.